Dockerfile

```diff
@@ -4,24 +4,36 @@ FROM python:3.8-alpine as base
 
 FROM base as builder
 
-WORKDIR /install
-
 RUN apk update && \
   apk add --virtual build-deps make git g++ python3-dev musl-dev jpeg-dev zlib-dev libevent-dev file-dev libffi-dev openssl && \
   apk add postgresql-dev
 # PDF Generation: weasyprint (libffi-dev jpeg-dev already included above)
 RUN apk add --virtual gdk-pixbuf-dev
 
-ADD requirements.txt /requirements.txt
-ADD requirements /requirements/
+ENV POETRY_HOME=/opt/poetry \
+    POETRY_VIRTUALENVS_IN_PROJECT=true \
+    POETRY_EXPERIMENTAL_NEW_INSTALLER=false \
+    POETRY_NO_INTERACTION=1
+    
+ENV PATH="$POETRY_HOME/bin:$PATH"
+
+RUN wget -O - https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+WORKDIR /opt/pysetup
 
-RUN pip install --prefix=/install --no-warn-script-location -r /requirements.txt
+ADD pyproject.toml ./
+ADD poetry.lock ./
+
+RUN poetry install --no-root --no-dev
 
 ####
 
 FROM base
 
-COPY --from=builder /install /usr/local
+COPY --from=builder /opt/pysetup/.venv /opt/pysetup/.venv
```

Comment:
Can we move it to /usr/local like before. Philosophy of the final container is to be structured like it is in built with the required packages, so it's better for them to be in /usr/local

Reply:
Also, please add poetry install directory in the final container and in the PATH so we are able to install any ad hoc package on running container